_data/accessibility-tests/memorable-date.yml

```diff
@@ -0,0 +1,79 @@
+title: Memorable date
+component_status: pass
+test_items:
+# General tests
+  - summary: Fields have auto-populate capability.
+    summary_additional: When you tab into memorable date fields using a keyboard, there will be an option in the browser to auto populate commonly used personal data.
+    test_status: conditional
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.5
+  - summary: Memorable date element pixel size is larger than 24x24.
+    summary_additional: When you select any memorable date element to inspect and review component code, you can see the total width of the element is at least 24 pixels wide.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 2.5.8
+  - summary: Element label text and background meet color contrast requirements.
+    summary_additional: When you use ANDI or another color contrast analyzer on memorable date, all labels and text inside elements meet a 4.5:1 WCAG contrast ratio.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.3
+  - summary: Memorable date elements have proper contrast.
+    summary_additional: When you use a color contrast analyzer, the contrast between the outline of the select component and text inputs is at least 3:1 from their respective surrounding backgrounds.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.1
+    wcag_criterion: 1.4.11
+  - summary: Memorable date display is consistent on mobile devices.
+    summary_additional: When you view the memorable date component on mobile devices and tablets, it displays in both portrait and landscape orientation with no loss of functionality.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.2
+    wcag_criterion: 1.3.4
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct memorable date elements.
+    summary_additional: When you zoom in to 400%, the content is still visible on the page. There is also no need for horizontal scrolling.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.2
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Focus indicator is clearly visible around memorable date elements.
+    summary_additional: When tabbing through the memorable date, a visible indicator shows where the user is. 
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.4.7
+  - summary: Keyboard actions can access all memorable date functionality.
+    summary_additional: You can change month, day and year and perform all other functions in a memorable date element using only your keyboard.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.2
+    wcag_criterion: 2.1.1
```

Comment:
@amycole501 @amyleadem Should I update all the keyboard/accessibility tests to have `3.9.0` for version tested?

Reply:
Deferring to @amycole501 or @alex-hull here since I am not sure if all criteria has been retested in 3.9.0

Reply:
Ooh, it looks like all of them were retested on 8/15. Would that fall in the 3.9.0 release timeframe? If so, then I think it makes sense to mark it as such.

Reply:
If the last test was on 8/15/24, that would be the 3.8.2 release (Reference: [USWDS releases](https://github.com/uswds/uswds/releases)). 

@mahoneycm - I don't see evidence of a keyboard retest in 3.9.0 (I might just be missing it). Let us know if you think the spreadsheet needs to be updated.

Reply:
Leaving these as `3.8.2` then!

Reply:
@amyleadem @RachelCorsino  We updated the select element as a result of user testing in uswds/uswds#6028 which was included in `3.9.0`

My thinking was that since this test is about accessing all memorable date functionality with the keyboard, we improved and tested this more recently! 

This is a super small detail so may not be necessary, but it might be worth considering when we need to update the spreadsheet after changes are made / features are introduced

Reply:
@mahoneycm That makes sense. It is a valid point that we should stay as up-to-date as possible in our version numbers for these checklist items. I am just not seeing evidence that the accessibility team re-tested with the keyboard in that PR or in the spreadsheet (but again, I might just be missing it).